.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: test
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
```

Comment:
```suggestion
  pull_request:
      branches:
         types: [opened, reopened,  edited]
```

Reply:
Maybe you can

Reply:
I've put in a ~ which should hopefully make it a bit more obvious that it's an empty 'any' target